blueocean-dashboard/package.json

```diff
@@ -89,5 +96,34 @@
       "react-router@any",
       "redux-thunk@any"
     ]
+  },
+  "jest": {
```

Comment:
might be good to have this config in the external jest config file `jest.config.js`

Reply:
That is true, i would have to update the js-builder-jest plugin though. Which i might do in another pr